test/test_optim.py

```diff
@@ -798,6 +797,74 @@ def test_lambda_lr(self):
                              lr_lambda=[lambda x1: 0.9 ** x1, lambda x2: 0.8 ** x2])
         self._test(scheduler, targets, epochs)
 
+    def test_sgdr_lr1(self):
+        iters = 50
+        eta_min = 1e-10
+        T = 10
+        T_mult = 1
+        T_cur = 0
+        targets = [[0.05], [0.5]]
+        scheduler = SGDR(self.opt, T_0=T, T_mult=T_mult, eta_min=eta_min)
+        for i in range(1, iters, 1):
```

Comment:
var `i` is not used, maybe replace it with `_`? (same for other for loops in this file)